advFileControl_test.go

```diff
@@ -246,12 +243,11 @@ func BenchmarkADVFCFieldInclusionBlockCount(b *testing.B) {
 func testADVFCFieldInclusionEntryAddendaCount(t testing.TB) {
 	fc := mockADVFileControl()
 	fc.EntryAddendaCount = 0
-	if err := fc.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if !strings.Contains(e.Msg, msgFieldInclusion) {
+	err := fc.Validate()
+if !base.Match(err, ErrConstructor) {
```

Comment:
Is this format off?

Reply:
Yeah, I fixed the formatting issues in a separate commit